langchain/chat_models/vertexai.py

```diff
@@ -120,13 +150,16 @@ def _generate(
         history = _parse_chat_history(messages[:-1])
         context = history.context if history.context else None
         params = {**self._default_params, **kwargs}
+        examples = kwargs.get("examples", None)
+        if examples:
+            params["examples"] = _parse_examples(examples)
         if not self.is_codey_model:
             chat = self.client.start_chat(
                 context=context, message_history=history.history, **params
             )
         else:
             chat = self.client.start_chat(**params)
-        response = chat.send_message(question.content, **params)
+        response = chat.send_message(question.content)
```

Comment:
is this a backwards compatible change?

Reply:
yes, since we send the same params when we start the `chat` and we don't persist the ChatSession.

Reply:
nice!